manifests/master/tuning.pp

```diff
@@ -0,0 +1,62 @@
+class classroom::master::tuning (
+  $jruby_purge         = $classroom::params::jruby_purge,
+  $jvm_tuning_profile  = $classroom::params::jvm_tuning_profile,
+) inherits classroom::params {
+
+  # See https://tickets.puppetlabs.com/browse/PE-9704
+  if $jruby_purge {
+    $cert   = '/etc/puppetlabs/puppet/ssl/certs/pe-internal-classifier.pem'
+    $key    = '/etc/puppetlabs/puppet/ssl/private_keys/pe-internal-classifier.pem'
+    $cacert = '/etc/puppetlabs/puppet/ssl/certs/ca.pem'
+    $master = 'https://master.puppetlabs.vm:8140'
+    $api    = 'puppet-admin-api/v1/jruby-pool'
+
+    cron { 'purge jruby pool':
+      ensure  => 'present',
+      command => "curl -i --cert ${cert} --key ${key} --cacert ${cacert} -X DELETE ${master}/${api}",
+      minute  => ['0', '30'],
+      target  => 'root',
+      user    => 'root',
+    }
+  }
+
+  if $jvm_tuning_profile != false {
+
+    case $jvm_tuning_profile {
+      'lvm': {
```

Comment:
it will replace the file if the instructor enables it. Unfortunately, there's at least one setting that must be set prior to install so this isn't a complete replacement